kubernetes/services/sg_manager.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sg-configmap
+data:
+  application.properties: |
+    ignite.host=10.99.135.32
+    ignite.port=10800
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sgmanager
+  name: sgmanager
+spec:
+  selector:
+    matchLabels:
+      app: sgmanager
+  replicas: 1
+  #replicas: 3
+  template:
+    metadata:
+      labels:
+        app: sgmanager
+    spec:
+      volumes:
+        - name: sg-volume
+          configMap:
+            name: sg-configmap
+            items:
+              - key: application.properties
+                path: application.properties
+      containers:
+        - image: zhonghaolyu/repo:sgKube1
```

Comment:
We will need to add one more step of pushing container image to the "official" container registry in the CI workflow (for every PR merge) so that this yaml file could pull the latest "official" image.

Create an issue #https://github.com/futurewei-cloud/alcor/issues/270 for tracking.

Reply:
Sure ~ when CI is finished, I'll change it